generators/php/sdk/src/endpoint/request/EndpointRequest.ts

```diff
@@ -25,7 +25,7 @@ export abstract class EndpointRequest {
     ) {}
 
     public getRequestParameterName(): string {
-        return this.context.getParameterName(this.sdkRequest.requestParameterName);
+        return `$${this.context.getParameterName(this.sdkRequest.requestParameterName)}`;
```

Comment:
i'd rather keep the convention that `get...Name` does not include `$`

in my mind, the `$` is not part of the name, so with the method name semantics, this makes sense

Reply:
Discussed offline - this is the only place where it's required since we manually specify the request variable name (without the AST helpers).